### PR TITLE
Added back to top button as per requirements

### PR DESCRIPTION
**Pull request by swamimalode07**
I have made the changes as per your requirements 
Added a back to top button from lucide.dev and ensured proper styling above the fullscreen button 

Can you please assign it Hactoberfest-accepted and gssoc-extd level 2 or 3

thanks

### CODE REVIEW

**Comment by swamimalode07** [comment]
image
Here is the screenshot

File: .env.example
Comment (moo-light) [inline .env.example:1]: i saw something not supposed to be here

Comment (swamimalode07) [reply .env.example:1]: Can you explain me what's wrong

Comment (moo-light) [reply .env.example:1]: Here 
image
the changes in `.env.example` have some magic string in it `pnpm dev`

Comment (swamimalode07) [reply .env.example:1]: can you please explain me , whats wrong i literally have no idea

Comment (moo-light) [reply .env.example:1]: you added a random `pnpm dev` into `.env.example` 
image

maybe it's okay for the whole project when it is deployed but 
it's not clean

**Pull request closed without merging**